charts/kafka-lag-exporter/templates/060-ServiceMonitor.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: monitoring.coreos.com/v1
```

Comment:
Since this CRD is only installed with the Prometheus Operator it would make sense to only deploy a `ServiceMonitor` if the user requests it.  Can you add configuration to `values.yaml` to create this resource, with a default of `false`?

Reply:
Added a option `.Values.prometheus.serviceMonitor.enabled` to enable ServiceMonitors, disabled by default